server/led_sign/led_sign_server.py

```diff
@@ -14,6 +15,7 @@ class LedSignServicer(led_sign_pb2_grpc.LedSignServicer):
     CURRENT_DIRECTORY = os.path.dirname(os.path.abspath(__file__)) + sep
     proc = None
     sign_data = {}
+    visual = LedSignMock()
 
     def WriteCommandToSign(self, request):
```

Comment:
Inside this method, you should call your `update_sign_visual` method to update the mock visual you have.

something like
```python
print(command)
.
.
.
<self.sign_data is defined here>
.
.
.
visual.update_sign_text(self.sign_data)
```